pre_commit/languages/rust.py

```diff
@@ -73,6 +74,8 @@ def _add_dependencies(
 ) -> None:
     crates = []
     for dep in additional_dependencies:
+        if dep.startswith(FEATURE_PREFIX):
+            continue
```

Comment:
I don't think this should be here at all?

Reply:
You're right, the features never reach this function.